README.md

```diff
@@ -42,8 +42,8 @@ This repository is administered by Benjamín J. Sánchez ([@BenjaSanchez](https:
 ### Installation Instructions
 * For users: Clone it from [`master`](https://github.com/SysBioChalmers/yeast-GEM) in the Github repo, or just download [the latest release](https://github.com/SysBioChalmers/yeast-GEM/releases). If you work in python, please create an environment with all requirements:
   ```bash
-  pip install -r requirements.txt     # installs all dependencies
-  type nul > .env                     # creates a .env file for locating the root
+  pip install -r requirements/requirements.txt  # installs all dependencies
+  type nul > .env                               # creates a .env file for locating the root
```

Comment:
I'm not sure what the intended output is, but maybe a simpler alternative would do the job?
```
touch .env
```

Reply:
Thanks for the suggestion! It achieves the same thing indeed. Done in [`02032da`](https://github.com/SysBioChalmers/yeast-GEM/pull/235/commits/02032da6cffd75250a185cf971791378449b564e)